e2e/utils/create-project-utils.ts

```diff
@@ -81,7 +81,7 @@ export function newProject({
       packageInstall(packages.join(` `), projScope);
 
       // stop the daemon
-      execSync('nx reset', {
+      execSync(`${getPackageManagerCommand().runNx} reset`, {
```

Comment:
This should be already addressed by another merged PR, but good catch regardless

Reply:
This PRs rebased and still needs the change. 🤷